src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  transition: transform ease 3s;
+  &:hover {
+    transform: scale(1.2);
+  }
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 36px;
+    padding-left: 3px;
+    padding-right: 3px;
+  }
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    margin-bottom: 5px;
+    color: #060b35;
+    line-height: 18px;
+  }
+  &:hover .card__title {
+    color: #34568b;
+  }
+  &__code {
+    font-size: 10px;
+    color: #616070;
+    margin-bottom: 18px;
+  }
+  &__feedback {
+    width: 166px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 22px;
+  }
```

Comment:
The width of the feedback section is hard coded. It's better to let it take the necessary space by itself, so remove the width.